src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -114,7 +114,7 @@ public boolean unlock(String datasetName, String flightId) {
      * @throws SQLException
      */
     @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
-    public UUID createAndLock(Dataset dataset, String flightId) throws SQLException {
+    public UUID createAndLock(Dataset dataset, String flightId) throws Exception {
```

Comment:
Minor: How come the switch to Exception? I'd prefer a more precise list.

Reply:
Just to make it compile, so I could run tests. Will switch them back.